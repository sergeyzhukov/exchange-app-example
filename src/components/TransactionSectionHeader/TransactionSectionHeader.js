import React from 'react'
import { View, Text, StyleSheet } from 'react-native'

export default ({ title }) => (
  <View style={styles.container}>
    <Text style={styles.title}>{title}</Text>
  </View>
)

const styles = StyleSheet.create({
  container: {
    height: 24,
    justifyContent: 'center',
    backgroundColor: '#F0F1F2',
    paddingHorizontal: 16,
    borderBottomColor: '#C8C7CC',
    borderBottomWidth: StyleSheet.hairlineWidth,
  },
  title: {
    fontSize: 12,
    color: '#343434',
  },
})
