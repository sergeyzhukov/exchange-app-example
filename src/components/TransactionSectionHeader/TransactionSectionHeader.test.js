import React from 'react'
import renderer from 'react-test-renderer'
import TransactionSectionHeader from './TransactionSectionHeader'

test('renders correctly', () => {
  const component = <TransactionSectionHeader />
  const tree = renderer.create(component).toJSON()
  expect(tree).toMatchSnapshot()
})
