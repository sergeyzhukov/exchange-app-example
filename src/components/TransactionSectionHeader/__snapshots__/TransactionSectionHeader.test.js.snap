// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#F0F1F2",
      "borderBottomColor": "#C8C7CC",
      "borderBottomWidth": 0.5,
      "height": 24,
      "justifyContent": "center",
      "paddingHorizontal": 16,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#343434",
        "fontSize": 12,
      }
    }
  />
</View>
`;
